crates/typst-cli/src/main.rs

```diff
@@ -40,6 +41,11 @@ fn main() -> ExitCode {
         Command::Watch(command) => crate::watch::watch(command),
         Command::Query(command) => crate::query::query(command),
         Command::Fonts(command) => crate::fonts::fonts(command),
+        #[cfg(not(feature = "self-update"))]
+        Command::Update(_command) => {
+            Err("updating is disabled for this distributable".into())
```

Comment:
```suggestion
            Err("Self-updating is not enabled for this distribution, please update with the same package manager or mechanism used for initial installation.".into())
```

Reply:
I propose `self-updating is not enabled for this executable, please update with the package manager or mechanism used for initial installation`. We generally keep everything lowercase and the final period is added by the error printing code.